core/src/main/java/org/mule/runtime/core/util/queue/objectstore/DefaultQueueInfoDelegate.java

```diff
@@ -1,130 +0,0 @@
-/*
- * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
- * The software in this package is published under the terms of the CPAL v1.0
- * license, a copy of which has been included with this distribution in the
- * LICENSE.txt file.
- */
-package org.mule.runtime.core.util.queue.objectstore;
-
-import java.io.Serializable;
-import java.util.Collection;
-import java.util.LinkedList;
-
-/**
- * The default QueueStoreDelegate. This uses a LinkedList to store the members of the queue.
- *
- * @deprecated this class will be removed in Mule 4.0 in favor of the new queue implementation
```

Comment:
As long as I know there is no replacement for queues... are you sure we can remove them?